plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -542,6 +543,39 @@ private static function get_orders_by_gateway() {
 		return $orders_by_gateway_currency;
 	}
 
+	/**
+	 * Get orders origin details.
+	 * 
```

Comment:
Minor/coding standards (no trailing spaces):

```suggestion
	 *
```